tests/foreman/cli/test_satellitesync.py

```diff
@@ -1398,117 +1320,6 @@ def test_negative_import_invalid_path(self, module_org):
             '--metadata-file option'
         ) in error.value.message
 
-    @pytest.mark.tier2
-    def test_negative_export_cv_with_on_demand_repo(
-        self, export_import_cleanup_module, target_sat, module_org
-    ):
-        """Exporting CV version having on_demand repo throws error
-        with "fail-on-missing-content" option set
-
-        :id: f8b86d0e-e1a7-4e19-bb82-6de7d16c6676
-
-        :steps:
-
-            1. Create product
-            2. Create repos with immediate and on_demand download policy
-            3. Sync the repositories
-            4. Create CV with above product and publish
-            5. Attempt to export CV version contents
-
-        :expectedresults:
-
-            1. Export fails with error and no export is created
-
-        :BZ: 1998626, 2067275
```

Comment:
Both BZs look like a NOTABUG, but isn't this test valid?

Reply:
I see, if I'm getting it right, you merging both positive and negative scenarios, then maybe add this BZ numbers to above test?

Reply:
Yes, this test case is valid and has been merged with the test above. In the above test I kept just the 1998626 (L1154), which was CLOSED WONTFIX, while 2067275 was CLOSED NOTABUG, so I removed it.

But the more I think of it I believe I should remove 1998626 too, because this test does not exercise any fix of this CLOSED WINFIX bug, it only uses some workaround... :thinking: